production/scrapers/texas.R

```diff
@@ -1,6 +1,10 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
+texas_check_date <- function(x, date = Sys.Date()){
+    NULL
```

Comment:
how come this is null?

Reply:
good point When i tried doing these the API was down ill check again!

Reply:
They dont have dates :(